### ISSUE TITLE
RTM Poke

### ISSUE THREAD

**Issue by smudge202** [herecydev/TestAttributes#39]
Was just creating some RTM tests and thought I'd give this a poke. Looks like it doesn't use the [xUnit RTM flavours](https://xunit.github.io/docs/getting-started-dotnet-core.html); wondered if you'd be so inclined as to pour in a little TLC?

**Comment by herecydev** [comment]
Yep, it needs to also support `netstandard` for RTM to work so it's definitely a blocker. I'm more than happy for a PR if you have time otherwise I will get it going in the near future.
